server/index.js

```diff
@@ -23,14 +24,19 @@ app.use('/test', (req, res) => {
   res.send('3-2-1 testing! Server is serving!');
 });
 
+// app.get('*', (req, res) => {
+//   res.sendFile('index.html', { root: __dirname })
+// })
```

Comment:
I can see how this could be redundant to the route starting on line 31, so I understand not needing it. However, if it's not in use, can probably delete this commented text.

Reply:
Deleted.